cms/djangoapps/contentstore/utils.py

```diff
@@ -1948,6 +1976,7 @@ def get_container_handler_context(request, usage_key, course, xblock):  # pylint
         # Status of the user's clipboard, exactly as would be returned from the "GET clipboard" REST API.
         'user_clipboard': user_clipboard,
         'is_fullwidth_content': is_library_xblock,
+        'sequence_ids': sequence_ids,
```

Comment:
[nit] rename to `aggregated_sequence_ids`
Because of we need whole sequences in course, not just depending to this unit

Reply:
Agree, the name can be more specific like suggested

Reply:
Agree on renaming, I would go with `course_sequence_ids`